src/app.js

```diff
@@ -1 +1,38 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function copyFileToDestination() {
+  const [sourcePath, destinationPath] = process.argv.slice(2);
+
+  if (process.argv.length < 4) {
+    console.log('Please enter file name and destination to copy');
+
+    return;
```

Comment:
Maybe we could give user one more attempt to write a file name and destination correctly? Because right now we have an error... aaand it just stuck, let's give user a question again.